eloquent/distribution.yaml

```diff
@@ -970,16 +970,16 @@ repositories:
     doc:
       type: git
       url: https://github.com/flynneva/grbl_ros.git
-      version: eloquent
+      version: main
     release:
       tags:
         release: release/eloquent/{package}/{version}
       url: https://github.com/flynneva/grbl_ros-release.git
-      version: 0.1.1-1
+      version: 0.0.14-1
```

Comment:
Same question as https://github.com/ros/rosdistro/pull/27079/files#r509860036

Reply:
same answer - yeah its fine. no one is really using this package yet since the previous released version does not function as desired - it cant control a grbl cnc really